File: app/src/main/java/com/washfi/todoapp/DetailActivity.java
package com.washfi.todoapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

import static com.washfi.todoapp.AppConstant.DESCRIPTION;
import static com.washfi.todoapp.AppConstant.TITLE;

public class DetailActivity extends AppCompatActivity {

    TextView textViewTitle, textViewDescription;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindView();
        setUpIntentData();
    }

    private void setUpIntentData() {
        Intent intent = getIntent();
        textViewTitle.setText(intent.getStringExtra(TITLE));
        textViewDescription.setText(intent.getStringExtra(DESCRIPTION));
    }

    private void bindView() {
        textViewTitle = findViewById(R.id.textViewTitle);
        textViewDescription = findViewById(R.id.textViewDescription);
    }
}

File: app/src/main/java/com/washfi/todoapp/DetailActivity.kt
package com.washfi.todoapp

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class DetailActivity : AppCompatActivity() {
    val TAG ="DetailActivity"
    lateinit var textViewTitle:TextView
    lateinit var textViewDescription:TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindView()
        setUpIntentData()
    }

    private fun bindView() {
        textViewTitle = findViewById(R.id.textViewTitle)
        textViewDescription = findViewById(R.id.textViewDescription)
    }

    private fun setUpIntentData() {
        val intent = intent
        val title = intent.getStringExtra(AppConstant.TITLE)
        val description = intent.getStringExtra(AppConstant.DESCRIPTION)
        textViewTitle.text = title
        textViewDescription.text = description
    }
}